parallel.gemspec

```diff
@@ -7,6 +7,12 @@ Gem::Specification.new name, Parallel::VERSION do |s|
   s.authors = ["Michael Grosser"]
   s.email = "michael@grosser.it"
   s.homepage = "https://github.com/grosser/#{name}"
+  s.metadata = {
+    "bug_tracker_uri"   => "https://github.com/grosser/#{name}/issues",
+    "documentation_uri" => "https://www.rubydoc.info/gems/#{name}/#{s.version}",
```

Comment:
there are no docs there ... prefer to leave this empty

Reply:
🤔 I wonder why the readme isn't rendered on that page.

Reply:
it's not rdoc or so ... but I don't want my docs there anyway, I want them close to the source so people make PRs :)

Reply:
Other gems readme.md get rendered. Take Spring for example: 
https://www.rubydoc.info/gems/spring
https://github.com/rails/spring/blob/master/README.md

I wonder if it's the capitalisation. `Readme.md` vs `README.md`.